lib/sidekiq/job/iterable.rb

```diff
@@ -273,6 +273,11 @@ def build_enumerator(params, cursor:)
         end
       end
 
+      def should_interrupt?
```

Comment:
Can this method be considered a public feature of iterable jobs now? I was previously overwriting `interrupted?` to add a custom per-job max runtime. This one makes it much easier. 👍🏼

Reply:
If it's a private method, it is not public.